### ISSUE TITLE
markdown render incorrectly in operation_semantics#batchnormgrad

### ISSUE THREAD

**Issue by mingting** [tensorflow/tensorflow#50730]
## URL(s) with the issue:

https://www.tensorflow.org/xla/operation_semantics#batchnormgrad

## Description of issue (what needs changing):

the brachnormgrad table render incorrectly

### Clear description

![image](https://user-images.githubusercontent.com/1769997/125224912-fb446200-e300-11eb-9b90-39edc743582f.png)

### Correct links

no find yet

markdown to html used google internel doc system.

**Comment by ymodak** [comment]
@mingting This is the link https://github.com/tensorflow/tensorflow/blob/master/tensorflow/compiler/xla/g3doc/operation_semantics.md you may want to fix.

**Comment by mingting** [comment]
@ymodak yes. 
I tried to contribute for this markdown file, but I don't know which table syntax this markdown used.
I discussed with community,
https://discuss.tensorflow.org/t/question-about-markdown-table-format/2492

I get know, because of the markdown used a google syntax,  developer out of google cannot change this markdown file.

the markdown to html should take care google syntax well.

but for this issue, theere is problem in html file. I think this should be bug of markdown to html system.
